internal/constants/get_daytona_script.go

```diff
@@ -54,7 +54,7 @@ esac
 
 DOWNLOAD_URL="$BASE_URL/$VERSION/daytona-$FILENAME"
 
-echo -e "\nDownloading server from $DOWNLOAD_URL"
+echo -e "\nDownloading Daytona binary from $DOWNLOAD_URL"
 
 # Create a temporary file to download the server binary. Just in case the user
```

Comment:
@tarunrajput there are still references to the "server binary" in the code. Please rename that as well.

Reply:
done